api/graphql/makeModels.js

```diff
@@ -170,7 +220,19 @@ export default function makeModels (userId, isAdmin, apiClient) {
       },
       relations: [
         'memberships',
-        'moderatedGroupMemberships',
+        {
+          membershipCommonRoles: {
+            querySet: true,
+            filter: (relation, { groupId }) => {
+              return relation.query(q => {
+                if (groupId) {
+                  q.where('group_id', groupId)
+                }
+              })
+            }
+          }
+        },
+        'moderatedGroupMemberships', // TODO: still need this?
```

Comment:
There are a few uses of this in Mobile. Did you refractor them out of EVO?

Reply:
I did, through just realized I hadn't removed it from all the queries yet